repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,23 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
+      {{#parameters}}
+        {
+          "key": "{{key}}",
+          "value": "{{value}}"
+        },
+      {{/parameters}}
+        {
+          "key": "label",
+          "value": "created_by=marathon"
```

Comment:
I think there's a chance these labels may collide with what a user may define. Can we instead use something less likely for a user to define such as
```
"value": "universe_dummy_label_key=dummy_value"
```